frontend/src/components/AllocationTopbar/AllocationTopbar.js

```diff
@@ -11,10 +11,8 @@ const rawIcon = <FontIcon className='material-icons'>code</FontIcon>
 class _AllocationTopbar extends PureComponent {
 
   handleActive (tab) {
-    let path = location.pathname.split('/')
-    path.pop()
-    path = path.join('/')
-    this.props.router.push(path + '/' + tab)
+    const prefix = `/nomad/${this.props.router.params.region}/allocations/${this.props.allocation.ID}`
```

Comment:
Alright, this all needs to be quoted properly otherwise allocations or region with a slash inside will break majestically :)

(like for example, instance of a periodic job :) )